lib/webidl2.js

```diff
@@ -851,7 +851,7 @@
       return ret;
     }
 
-    function interface_rest(isPartial, typeName = "interface") {
+    function interface_rest(partialModifier = null, typeName = "interface") {
```

Comment:
I'd really prefer if this was:
```JS
interface_rest(typeName = "interface", { partialModifier = null } ) {
```
It kills the annoying partialModifier boolean/null trap.